### DISCUSSION TITLE
PCB's available for sale?

### DISCUSSION THREAD

**Discussion by bjoerns1983** [dokuhn/openDTU-BreakoutBoard#15] [Q&A] [answered]
Hi there,
i love your idea, because i was just about designing the same but as far not as professional as yours because i have never done that before.
Just have two questions:
1. Dow you have some boards for sales, i would need 3 units.
2. Would is be possible possible to change the position of the cap so that also higher ones could be mounted
3. In some forums i have read that beside the elctrolyte cap also a ceramic one should be place infront of the NRF.

**Comment by dokuhn** [comment]
Hey, thank you for your positive feedback. 

Regarding 1): I currently have three unassembled boards left. I can give two of them to someone, one I need as a reserve. In my network (friends and colleagues) there are now 6 of these small units in the field.

Regarding to 2) and 3): Usually many electrolytic capacitors with 10uF in the lower voltage range fit exactly in this position. 100V resistant electrolytic capacitors often build higher, as you are right. But it is no problem to change the position in one of the next commits/pushs. Larger changes like adapting to existing cases etc. I will maintain in a second branch.

**Comment by bjoerns1983** [comment]
I would to take the two board you have left over, or is there a gerber file etc. which only needs to be uploaded to jlpcb etc. to get them manufactured.
Sorry if the question might be dumb i have never done that before.

Really love the simplicity of the adapter.

**Comment by dokuhn** [comment]
I can send you two boards and the right capacitors for some small bucks. If necessary I will order more boards soon. That depends on the requirements in my network with friends and colleagues.

**Comment by bjoerns1983** [comment]
That would be awesome, simply tell me what you get incl. shipping in Germany.
Are there private messages on Github so would could clearify the details?

**Comment by dokuhn** [comment]
You can send me a mail at [dominik.kuhn90@googlemail.com](mailto:dominik.kuhn90@googlemail.com), then we can coordinate details about shipping etc.

**Comment by dirkheiden** [comment]
It’s not very difficult to order these PCB from i.e. Aisler. The capacitors can be ordered on eBay for example.
![227C68A8-05B3-4CB2-858A-DFFDC7F278E6](https://user-images.githubusercontent.com/44818215/204105029-52d3fde3-2ead-4d4e-92cf-cfa9416a1e17.jpeg)

**Comment by bjoerns1983** [comment]
Okay after waiting for ages for the Pinheaders it done:
![IMG_7939](https://user-images.githubusercontent.com/424566/205958708-0588c5ce-badb-48d2-83fa-452bfce7e834.JPG)

**Comment by bjoerns1983** [comment]
~~If anybody is interested: I have ordered some boards myself and have some left over.~~

**Comment by moritzlerch** [comment]
Old vs. New

Thanks to @bjoerns1983 for the PCB 😄 

![image](https://user-images.githubusercontent.com/59474941/209443721-5197d0f0-cf3e-44db-872e-e471ec2d2944.jpeg)

**Comment by bjoerns1983** [comment]
Okay all gone here.

**Comment by evilmumi** [comment]
Hi, are some PCB still available to buy?
Or can some1 tell me where / how i can order this best from .de
I only need one

**Comment by Archibald-Harry-Tuttle** [comment]
I have some left.
Added mounting holes on all 4 corners for 3mm screws.
just write me pn pls.

**Comment by evilmumi** [comment]
> I have some left. Added mounting holes on all 4 corners for 3mm screws. just write me pn pls.

Hi, a pn on github is not possible or?

**Comment by Archibald-Harry-Tuttle** [comment]
> > I have some left. Added mounting holes on all 4 corners for 3mm screws. just write me pn pls.
> 
> Hi, a pn on github is not possible or?

You find me in the discord server of OpenDTU repository. 
Added comment:
there i not found a link to the 
"hoymiles research" discord server.

Here you will find it:
https://github.com/lumapu/ahoy

**Comment by guha7** [comment]
Does anybody have one PCB for following setup left?
- Espressif ESP32 WLAN Dev Kit Board Development Bluetooth Wifi v1 WROOM32 Node MCU 
https://www.makershop.de/plattformen/nodemcu/espressif-esp32-dev-kit-board/
![ESP32-Pinout](https://user-images.githubusercontent.com/122552787/212337518-1fd03a52-9f5f-4fc2-adca-7947b88b984b.png)

- NRF24L01+ PLUS - PA LNA SMA Antenne Long Range Funk Modul Raspberry Arduino 
https://www.makershop.de/module/funk/nrf24l01-pa-lna-sma-antenne/
![Pinout-nRF24L01-PA-LNA-External-Antenna-Wireless-Transceiver-Module](https://user-images.githubusercontent.com/122552787/212337492-37d82e0a-503b-42d2-878d-d9e6cfc927c1.png)

Thanks for your offer!

**Comment by Archibald-Harry-Tuttle** [comment]
I have some left.
like in the last post before,
you find me on discord.

**Comment by webwerb** [comment]
Sorry, I am too stupid to find the contact person ....

I need a total of 3 boards for the following configuration

- Espressif ESP32 WLAN Dev Kit Board Development Bluetooth Wifi v1 WROOM32 Node MCU
- NRF24L01+ PLUS - PA LNA SMA Antenna Long Range Radio Module Raspberry Arduino
- 0.96 inch OLED display I2C SSD1306
-
Ev. also 3 enclosures for use with display, as I don't have a printer yet.

Would be grateful for appropriate tips for a beginner here on the platform.

Thanks!

**Comment by dokuhn** [comment]
Hi, you are not too stupid to find the contact person. 

Thank you for your request. I am the contact person for this repo, but as you may have seen, this is a community driven project. That means members of the community support each other with for example boards or even questions about variants etc.. In the "home" README file is described for which board this breakout board is intended. Is your board compatible with it ? There are also other variants with less pins which can be used by a small hack of the board. For this you can have a look at the corresponding issue. Otherwise @Archibald-Harry-Tuttle  as you can see above probably still board left. You can reach him under the following [Discord link](https://discord.gg/WzhxEY62mB). I currently have no more boards left, new ones are on the way and will take more than 10 days to arrive.

Dominik Kuhn

**Comment by Pixel-Chris** [comment]
@dokuhn 
Hi, sind die Boards reinzufällig schon angekommen?

**Comment by webwerb** [comment]
Moin,

 

gerade eingetroffen.

 

Super schnell – Danke!

 

Ich Dussel hab natürlich vorher nicht geschaut ob 30 oder 38 Pin und ich hab natürlich die 30 Pin Version vom ESP32 ….

 

Das wird dann so nicht passen?

 

Also 30 Pin Borads kaufen (wo?) oder 38 Pin ESP 32 kaufen. Dann könnte ich gleich welche mit externer Antenne fürs WLAN nehmen.

 

Grüße

 

Uwe

 

Von: Pixel-Chris ***@***.*** 
Gesendet: Donnerstag, 26. Januar 2023 10:04
An: dokuhn/openDTU-BreakoutBoard ***@***.***>
Cc: webwerb ***@***.***>; Comment ***@***.***>
Betreff: Re: [dokuhn/openDTU-BreakoutBoard] PCB's available for sale? (Issue #2)

 

@dokuhn <https://github.com/dokuhn> 
Hi, sind die Boards reinzufällig schon angekommen?

**Comment by dokuhn** [comment]
Nachdem in Fernost auch das neue Jahr vorbei ist, hab ich eine Mail bekommen, dass die nächsten Platinen unterwegs sind zu mir. Sprich die 20 Platinen des normalen Breakout-Board werden demnächst dann eintreffen. Die SMD Variante ist in den letzten Zügen und wird in den nächsten Wochen ebenfalls als Prototypen mal aufgebaut.

**Comment by dirkheiden** [comment]
Auf die SMD Version freue ich mich sehr. Hast Du dafür schon eine BOM?

**Comment by moritzlerch** [comment]
Vielleicht dann https://github.com/openscopeproject/InteractiveHtmlBom verwenden, dann wird die Bestückung echt kinderleicht 😁

**Comment by wiedimot** [comment]
> 

@dokuhn wie kann man die Boards denn bei dir bestellen?
Ich hätte auch interesse an 3 Stück. In dem Fall die "normale" Version des Breakout Boards.

**Comment by dokuhn** [comment]
@wiedimot aktuell hab ich von der normalen Variante noch gut 15 Stück "vorrätig". Du kannst du mir einfach eine persönliche Mail schreiben. Darin kannst du mir dann auch die Adresse etc. mitteilen wo das Ganze hin soll. In der Mail können wir dann klären ob du mir etwas überweist oder den PayPal Button nutzen möchtest. 

Auch wenn meine Platinen bei mir mal temporär aus sind, ist es so, dass die Community sich da sehr behilflich ist. Ich werde entsprechenden Link die Tage mal erweitern in der README damit die ganzen Channel auch gefunden werden können.

**Comment by andreas-bulling** [comment]
Hallo zusammen - ich hätte auch Interesse an einem Bord. Die normale Version würde mir reichen - sind noch welche da?

**Comment by dokuhn** [comment]
Wie in der Message zuvor beschrieben bitte ein persönliche  E-Mail an mich, dann können wir Details wie Adresse etc. dort auf dem kurzen Weg klären. 

### Kurzes Update:  
Aktuell sind noch rund **6-7 Boards** übrig.  Ich würde nochmal welche Bestellen. Ansonsten bitte nicht wundern, ich besorge jetzt keine hunderte der Boards in der Phase, in der ich Beschaffungen tätige für die SMD/SMT Variante. Aus diesem Grund gibt es meiner Seitz lediglich Losgrößen zwischen 20 - 50 Platinen.

**Comment by huckes** [comment]
Hallo, 
hatte auch Interesse an drei Boards per PM vor ein paar Tagen bekundet, leider bisher keine Antwort. Mail ist evtl. untergegangen oder ich bin zu ungeduldig?

Update: Dominik war verhindert, Platinen sind bestellt. Danke !!

**Comment by dokuhn** [comment]
### **Update:**

Die letzte Bestellung ist jetzt mehr oder weniger kommplett raus. Sprich das letzte BreakoutBoard liegt schon im Hausgang für den Weg zur Post. Ich danke allen für die Unterstützung des Projekts an dieser Stelle. Die nächsten Boards sind aktuell wohl in der Produktion und werden vorausichtlich Ende nächster Woche eintreffen. 

The last order is now more or less completely shipped. The last BreakoutBoard is already in the corridor on its way to the post office. I would like to thank everyone for their support of the project. The next boards are currently in production and will probably arrive at the end of next week.

**Comment by dokuhn** [comment]
### **Update:**

Die neue Variante ist mal prototypisch aufgebaut und wird getestet. Auch wenn es inzwischen schon merklich länger hell ist abends, könnte das Bild besser sein um 21 Uhr. ;-) 

![75dc7fadcf4748e5a89d798d3fc50fgg6](https://user-images.githubusercontent.com/16343380/225755693-e49cba51-2794-4c05-8abb-3d6ebf3b736d.jpeg)

**Comment by andy5macht** [comment]
Kannst du schon abschätzen ob du die SMD-Version (fertig bestückt) anbieten kannst und ob ich dann davon 1-2 Stück abhaben darf?
Würde mich freuen!

**Comment by dokuhn** [comment]
Ich freue mich das die neue Variante auf Interesse stößt. In Summe sind jetzt 4 Boards aufgebaut und werden durch unterschiedliche Kollegen der Community und im Umkreis des Vereins auf Herz und Nieren getestet. Danach (in den nächsten 1 - 2 Wochen) wird es ein Batch geben welches wir fertig bestückt bei Aisler und Co. Beschaffen. ![image](https://user-images.githubusercontent.com/16343380/226735700-275ebfbe-1f3d-477a-99dc-edc7f888dbbe.jpeg)

**Reply by frosty-geek** [reply]
Hi,

Ich hätte Interesse an 4 für unsere lokale Bubble. Wo kommt man da aktuell so kostentechnisch raus? 

Viele Grüße
Ralf

**Comment by mark82436** [comment]
Hallo Dominik,

ich hätte auch Interesse an 2 Stk. von den neuen Boards hast du da schon einen ungefähren Horizont wann, die soweit fertig sind?
Ist der Displayanschluss der J4? Wenn ja, wird dieser noch bestückt?

Gruß
Markus

**Comment by swingstate** [comment]
@bjoerns1983 wird die neue Board Version mit diesem Projekt kompatibel sein? https://github.com/helgeerbe/OpenDTU-OnBattery

Ich hätte Interesse an einem Board mit externem Antenneanschluss. Da ich aufgrund einer Augenkrankheit nicht gut/kaum gucken kann sind Lötarbeiten etc. für mich nicht möglich. 
Die Rev. 2.0 scheint dies zu adressieren.

**Comment by dokuhn** [comment]
Ich freue mich nach wie vor über eure Reaktion muss ich sagen. Die Tests sind im vollen Gange und auch die MicroSD Card funktioniert. Aktuell schreibe ich die wichtigsten Informationen in eine CSV Datei um diese offline auswerten zu können. Klar viele Nerds haben einen Raspi oder ähnliches mit MQTT und einer SmartHome Umgebung in der Ecke liegen. Aber eben nicht jeder, wie bspw. meine Eltern. Um aber auch diesen Personen die Möglichkeit zu bieten "Langzeitdaten" zu erfassen haben wir diesem Board eine MicroSD gespendet. 

Ein kleiner Bug (siehe [Issue 14](https://github.com/dokuhn/openDTU-BreakoutBoard/issues/14)) ist behoben und wir bestellen diese Woche die ersten Boards bei Aisler oder Co. in der fertig bestückten Variante. Sprich eure Interessensbekundungen sind aufgenommen à la "Vorbestellung". Den aktuellen Preis können wir noch nicht final sagen, werden wir aber in wenigen Tagen erfahren.

**Reply by dirkheiden** [reply]
Hallo Dominik,

ich würde ebenfalls gerne 2-3 Stück der bestückten Variante nehmen. 

Gruß Dirk

**Reply by Pixel-Chris** [reply]
Hi, 
Ich würde ebenfalls gerne 3 Stück nehmen.

greets
PixelChris

**Reply by 67069** [reply]
Hallo,

ich habe Interesse an 2 Boards. 

Danke

Andreas

**Reply by andy5macht** [reply]
wie erwartet "darf" ich einem Kumpel ebenfalls ein DTU machen. Hätte also nach wie vor interesse an 2stk.
Freue mich auf die Auswertung/Datenablage auf der SD-Karte

**Comment by dokuhn** [comment]
So für das erste Test-Batch mit 15 Boards ist das Part Placement beim PCB-Hersteller inkl. Assembly ist gemacht und abgeschickt. Jetzt heißt es warten und auf die Post demnächst freuen. 

So the first test batch with 15 boards the part placement at the PCB manufacturer incl. assembly is done and ordered. Now we have to wait and look forward to the delivery very soon. 

![e66588bd172b40aabdeb88598cc76471](https://user-images.githubusercontent.com/16343380/229366771-6f237303-badc-4764-926c-6ac2f23ac587.png)

![ea1d1f8b43b347edbd55386cc545e0d3](https://user-images.githubusercontent.com/16343380/229366778-2b583fc3-ab49-459e-8996-217cdc63628b.png)

**Reply by marcogueth** [reply]
Hi Dominik,

vielen Dank für die super Arbeit! Ich würde gern eines der Rev.2 Boards bestellen. Ist aus deinem Batch noch eines übrig? 
viele Grüße
Marco

**Comment by brainbarry** [comment]
Hallo Dominik,
falls in der Rev.2 auch ein Display berücksichtigt ist würde ich auch 2 für diese ESP32 version benötigen:

![ESP32](https://user-images.githubusercontent.com/127986601/231504334-aef4e50c-b84e-4a26-bb8a-9a7a845e69b2.jpg)

schönen Dank dir und den anderen Projektunterstützern dass wir Unwissenden nicht im Regen stehen gelassen werden. ;-)

**Comment by Pixel-Chris** [comment]
Wird darauf auch die AhoyDTU lauffähig sein?

**Comment by thomic** [comment, marked as answer]
Please see here: https://github.com/dokuhn/openDTU-BreakoutBoard/discussions/23
